webpack/containers/Application/overrides.scss

```
@@ -75,3 +75,38 @@ html .pagination-pf-pagesize.btn-group {
     margin-left: 200px;
   }
 }
+
+@keyframes hideme {
+  0% {
+    opacity: 1;
+  }
+
+  10% {
+    opacity: 1;
+  }
+
+  30% {
+    opacity: 1;
+  }
+
+  80% {
+    opacity: 1;
+  }
+
+  90% {
+    opacity: 1;
+  }
+
+  100% {
+    opacity: 0;
+  }
+}
+
+.tooltip {
+  margin:auto;
+  animation-direction: normal;
+  animation-name: hideme;
+  animation-duration: 6s;
```

```suggestion
  animation-duration: 2s;
```
6 seconds is too long to wait!